.github/workflows/release-spec-prod.yml

```diff
@@ -19,19 +20,51 @@ jobs:
     secrets:
       api_bot_pat: ${{ secrets.API_BOT_PAT }}
 
-  generate-versioned-specs:
-    runs-on: ubuntu-latest
-    needs: [run-required-validations]
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
-      - name: Install Go
-        uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32
-        with:
-          go-version-file: 'tools/cli/go.mod'
-      - name: Run foascli split command
-        run: |
-          pushd tools/cli
-          make build
-          popd
-          ./tools/cli/bin/foascli split -s ./tools/cli/test/data/openapi-foas-dev.yaml --env prod
+  release:
+      runs-on: ubuntu-latest
+      needs: [run-required-validations]
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+        - name: Install Go
+          uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32
+          with:
+            go-version-file: 'tools/cli/go.mod'
+        - name: Run foascli split command
+          run: |
+            pushd tools/cli
+            make build
+            popd
+            ./tools/cli/bin/foascli split -s ./tools/cli/test/data/openapi-foas-dev.json --env prod -o ./openapi/v2/openapi.json
+            cp -rf "./tools/cli/test/data/openapi-foas-dev.json" "./openapi/v2.json"
+            
+            ./tools/cli/bin/foascli split -s ./tools/cli/test/data/openapi-foas-dev.yaml --env prod -o ./openapi/v2/openapi.yaml
+            cp -rf "./tools/cli/test/data/openapi-foas-dev.yaml" "./openapi/v2.yaml"
+        - name: Create Issue
+          if: ${{ failure() }}
+          uses: imjohnbo/issue-bot@572eed14422c4d6ca37e870f97e7da209422f5bd
+          with:
+            labels: failed-release
+            title: "(PROD) Release: `foascli split` command failed :scream_cat:"
+            body: See https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+            token: ${{ secrets.api_bot_pat }}
+
+        - name: Create Pull Request
+          uses: peter-evans/create-pull-request@v6
+          with:
+            token: ${{ secrets.GITHUB_TOKEN }}
+            title: "release: (PROD) Release OpenAPI Spec :rocket:"
```

Comment:
small suggestion: you could add a label to the PR to easily be able to filter out all automated PRs

Reply:
good point! I will add it

Reply:
updated